### PR TITLE
:sparkles: Add Yahoo support

### LINKED ISSUE #2: [Feature] Add Yahoo Support
Add yahoo support for images, videos, news and answers

### PR DESCRIPTION
**Pull request by ghost**
feat: Add Images, Videos, News and Answers search for Yahoo

You can execute the command for search in Yahoo Images, Videos, News and Answers.

Resolves: #2

### CODE REVIEW

File: .bash_aliases [hunk 1]
```diff
@@ -261,12 +261,48 @@ bing() {
...
+        "-a")
+            shift
+            for term in $@; do
+                search="$search%20$term"
+            done
+            echo "Yahoo Answers search for: $@"
+            xdg-open "https://answers.search.yahoo.com/search/answers?p=$search"
+            ;;
+        *)
+            echo "Searching on DuckDuckGo: $@"
```
Comment (saglaniharsh) [inline .bash_aliases]: This should be searching on Yahoo

File: .bash_aliases [hunk 2]
```diff
@@ -306,7 +342,7 @@ duckduckgo() {
             xdg-open "https://duckduckgo.com/?q=$search&iaxm=maps"
             ;;
         *)
-            echo "Searching on DuckDuckGo: $@"
+            echo "Searching on Yahoo: $@"
```
Comment (saglaniharsh) [inline .bash_aliases]: This should be searching on DuckDuckgo

**Review by saglaniharsh — COMMENTED** [review]
working fine

**Review by saglaniharsh — CHANGES_REQUESTED** [review]
Please fix the ones mentioned in the comment and also update the readme

**Comment by ghost** [comment]
Okay it's good, thank you !

File: .bash_aliases (hunk 2 above)
Comment (ashawe) [reply .bash_aliases:345]: This is still not fixed and add tick the todo checkboxes in the readme too

Comment (saglaniharsh) [inline .bash_aliases]: This is still not fixed and add tick the todo checkboxes in the readme too

**Review by saglaniharsh — CHANGES_REQUESTED** [review]
Fix Typographical error and add tick the todo checkboxes in the readme too

File: .bash_aliases (hunk 2 above)
Comment (saglaniharsh) [reply .bash_aliases]: This wasn't fixed

**Comment by ghost** [comment]
Normally it's good (excuse me for this bad PR, i dont know much things on PR) I think it's good ^^

File: README.md [hunk 3]
```diff
@@ -68,7 +68,18 @@ NOW YOU CAN USE THE SEARCH FEATURES!!
...
- `-yh`:  searches on yahoo
+ `-yh`:  searches on Yahoo
+> Available Sub Options for Yahoo:\
+                  `-i`: 
+                      searches on Yahoo images\
+                  `-n`: 
+                      searches on Yahoo news\
+                  `-v`: 
+                      searches on Yahoo videos\
+                  `-m`:
```
Comment (saglaniharsh) [inline README.md]: answers should be ```-a``` not ```-m```

**Review by saglaniharsh — CHANGES_REQUESTED** [review]
Make the changes below:

**Comment by saglaniharsh** [comment]
> 
> 
> Normally it's good (excuse me for this bad PR, i dont know much things on PR) I think it's good ^^

It is okay. No worries ^ _ ^

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: Oh yes excuse me x(

Comment (ghost) [reply README.md]: It's good

**Review by saglaniharsh — APPROVED** [review]

**Pull request merged**